src/e2e-tests/page-objects/page.js

```diff
@@ -15,4 +15,20 @@ export default class Page {
     open() {
         browser.url(this.path);
     }
+
+    isButtonDisplayed(button) {
+        return button.isDisplayed();
+    }
+
+    isButtonEnabled(button) {
+        browser.waitUntil(() => {
+            return button.getAttribute('checked').includes('true');
+        }, 5000);
+    }
+
+    waitForButtonToBeDisplayed(button) {
```

Comment:
This can be registered as a wide custom command. Please see: https://webdriver.io/docs/customcommands.html
And since it is an element custom command you have to register it for element scope

Note: if you register a custom command to the browser scope the command won't be accessible for elements. Likewise, if you register a command to the element scope, it won't be accessible at the browser scope:

Reply:
Updated.